components/dashboard/src/projects/ProjectSettings.tsx

```diff
@@ -156,13 +159,38 @@ export default function () {
                     </div>
                 </div>
             </div>
-            {BillingMode.canSetWorkspaceClass(teamBillingMode) && (
-                <SelectWorkspaceClass
-                    workspaceClass={project.settings?.workspaceClasses?.regular}
-                    enabled={BillingMode.canSetWorkspaceClass(teamBillingMode)}
-                    setWorkspaceClass={setWorkspaceClass}
-                />
-            )}
+            <div>
+                <h3 className="mt-12">Workspaces</h3>
+                <p className="text-base text-gray-500 dark:text-gray-400">
+                    Choose the workspace machine type for your workspaces.
+                </p>
+                {BillingMode.canSetWorkspaceClass(billingMode) ? (
+                    <SelectWorkspaceClass
+                        workspaceClass={project.settings?.workspaceClasses?.regular}
+                        setWorkspaceClass={setWorkspaceClass}
+                    />
+                ) : (
+                    <Alert type="message" className="mt-4">
```

Comment:
**suggestion(non-blocking)**: This is definitely out of the scope here, but going with the `bg-blue-800` background color for the dark mode for the alert components sounds like an improvement. Feel free to ignore this in this PR and leave this work to https://github.com/gitpod-io/gitpod/issues/9834.



| Using `bg-blue-700` | Using `bg-blue-800` |
|-|-|
| <img width="1440" alt="Screenshot 2022-12-06 at 7 10 47 PM (2)" src="https://user-images.githubusercontent.com/120486/205977546-fb1e8972-0358-4c4b-9b51-b570463522a5.png"> | <img width="1440" alt="Screenshot 2022-12-06 at 7 10 32 PM (2)" src="https://user-images.githubusercontent.com/120486/205977558-b7f5c902-cc38-4dc2-9156-2f7f95666528.png"> |

Reply:
Adjusted the blue background color in dark theme. 👍

Reply:
Thanks, @jankeromnes!